### PR TITLE
Update list of yum packages needed to install, separate pip3 packages

### PR DESCRIPTION
**Pull request by eric-simpson**
I have updated the list of yum/pip3 packages needed to install ML/AI matplotlib and juypter.

### CODE REVIEW

File: machinelearning/README.md [hunk 1]
```diff
@@ -29,10 +37,11 @@ python3-jupyter|Jupyter notebook support|install it directly by pip3 after libzm
...
 
 ## Contents of this repo.
-- **dbconn/loadcsv2db2.py**, this is a tool to import your data from csv to db2 on i. Or course, in most cases your data is already in db2.  NOTE, this script is using the non-journal database.
-- **dbconn/runsql.py**, this is a tool of python, which let you run a sql command through command line.
-- **homeCredit** , this is based on one kernel of a real data science competation on kaggle: [homeCredit]( https://www.kaggle.com/c/home-credit-default-risk). The kernel we used is : [start-here-a-gentle-introduction](https://www.kaggle.com/willkoehrsen/start-here-a-gentle-introduction). While the most difference from it is that we tried to use data in db2 as the source instead of csv stream file. This notebook would help you to understand how data be reterieved from db2 and used in a normal data science project. 
-- **sklearndemos**, a folder to store some sklearn demos which running well on IBM i platform.
+- **dbconn/loadcsv2db2.py**: this is a tool to import your data from csv to db2 on i. Or course, in most cases your data is already in db2.  NOTE, this script is using the non-journal database.
```
Comment (kadler) [inline machinelearning/README.md:40]:
```suggestion
- **dbconn/loadcsv2db2.py**: this is a tool to import your data from csv to db2 on i. Of course, in most cases your data is already in db2.  NOTE, this script is using the non-journal database.
```

File: machinelearning/README.md [hunk 2]
```diff
@@ -15,12 +15,20 @@ tk|TK package for GUI support of TCL|
...
 python3-scikit-learn|scikit-learn package|
-matplotlib|matplotlib package|install it directly by pip3
-libzmq|Zero-MQ library|not available yet
-python3-jupyter|Jupyter notebook support|install it directly by pip3 after libzmq available
+libzmq-devel|Zero-MQ library|
+freetype-devel|Freetype library|
+libjpeg-turbo-devel|JPEG library|
+zlib-devel|zlib development package|
+libffi-devel|FFI development package|
++ After yum packages are installed, we need to install following Python packages by pip3: `pip3 install <package name>`.
```
Comment (kadler) [inline machinelearning/README.md:26]:
```suggestion
+ After yum packages are installed, the following Python packages need to be installed using pip3: `pip3 install <package name>`.
```

File: machinelearning/README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+# ML on IBM i
 ## Intro
   This project is used to demonstrate the capacity of IBM i platform on ML/AI area. I encounge you to use, share or even add more sub-projects/notebook to this repo.  Any comments , suggestions or advisors are welcome. thanks.
 
-## How to setup your environment of ML on i
+## How to setup your environment of ML on IBM i
 + Here's where we can  [setup RPM](http://ibm.biz/ibmi-rpms
 ) env on i. 
-+ After setup the RPM env on i, we need to install following packages by yum. A [detail instruction](https://www.ibmsystemsmag.com/IT-Strategy/11/2019/How-to-Start-ml-on-IBM-i?utm_content=105130364&utm_medium=social&utm_source=twitter&hss_channel=tw-488711278
++ After setup the RPM env on i, we need to install following packages by yum: `yum install <package name>`. A [detailed instruction](https://www.ibmsystemsmag.com/IT-Strategy/11/2019/How-to-Start-ml-on-IBM-i?utm_content=105130364&utm_medium=social&utm_source=twitter&hss_channel=tw-488711278
```
Comment (kadler) [inline machinelearning/README.md:8]:
```suggestion
+ After setup the RPM env on i, the following packages need to be installed using yum: `yum install <package name>`. A [detailed instruction](https://www.ibmsystemsmag.com/IT-Strategy/11/2019/How-to-Start-ml-on-IBM-i
```

File: machinelearning/README.md [hunk 4]
```diff
@@ -15,12 +15,20 @@ tk|TK package for GUI support of TCL|
...
 python3-devel|python3 development package|
 python3-ibm_db|DBAPI support package for IBM i|
-python3-numpy|numpy package  |
+python3-numpy|numpy package|
 python3-scipy|scipy package|
 python3-scikit-learn|scikit-learn package|
-matplotlib|matplotlib package|install it directly by pip3
-libzmq|Zero-MQ library|not available yet
-python3-jupyter|Jupyter notebook support|install it directly by pip3 after libzmq available
+libzmq-devel|Zero-MQ library|
```
Comment (kadler) [inline machinelearning/README.md:21]:
```suggestion
libzmq-devel|Zero MQ library|
```

**Review by kadler — COMMENTED** [review]
It'd also be good to see db2 replaced with Db2 to match the new trademark.

File: machinelearning/README.md (hunk 1 above)
Comment (eric-simpson) [reply machinelearning/README.md]: ok

File: machinelearning/README.md (hunk 2 above)
Comment (eric-simpson) [reply machinelearning/README.md]: ok

File: machinelearning/README.md (hunk 3 above)
Comment (eric-simpson) [reply machinelearning/README.md]: ok

File: machinelearning/README.md (hunk 4 above)
Comment (eric-simpson) [reply machinelearning/README.md]: ok

**Review by kadler — APPROVED** [review]

**Pull request merged**